daemon-tests/tests/happy_path.rs

```diff
@@ -628,16 +627,14 @@ async fn open_cfd_is_refunded() {
 
 #[tokio::test]
 async fn taker_notices_lack_of_maker() {
-    let short_interval = Duration::from_secs(1);
-
     let _guard = init_tracing();
 
     let maker_config = MakerConfig::default()
-        .with_heartbeat_interval(short_interval)
-        .with_dedicated_port(35123); // set a fixed port so the taker can reconnect
+        .with_dedicated_port(35123)
```

Comment:
Not needed for this PR: 

I think it would be nice to use random ports here to avoid port-in-use conflicts.

Reply:
I've extracted #2308